src/authorizations/utils/permissions.ts

```diff
@@ -110,6 +112,8 @@ export const specificBucketsPermissions = (
   })
 }
 
+// assign permission string ('read' or 'write' is the second argument)
```

Comment:
(nit): should we leave this comments here?

Reply:
I agree with @Palakp41 that we do not need comments in here. But i will defer to @jrenee42 on this because she said it made it easier for her to understand.